README.md

```diff
@@ -48,7 +47,9 @@ our documentation, add new features—up to you! Thank you for pitching in.
 
 ## Compatibility
 
-Mail supports Ruby 1.8.7+, including JRuby and Rubinius.
+Mail supports Ruby 2.5+, including JRuby and TruffleRuby.
+
+As new versions of Ruby are released, Mail will be compatible with support for the "preview" and all "normal maintenance", "security maintenance" and the two most recent "end of life" versions listed at the [Ruby Maintenance Branches](https://www.ruby-lang.org/en/downloads/branches/) page.  Pull requests to assist in adding support for new preview releases are more than welcome.
```

Comment:
@deivid-rodriguez @voxik I added this paragraph, seem reasonable to you?  @jeremy do you have any input?

Reply:
Ah, alright, so we do keep Ruby 2.5 support, correct?

Reply:
It seems reasonable to me, by the way!

Reply:
That makes sense :+1: 

> Ah, alright, so we do keep Ruby 2.5 support, correct?

I read it the same way.